### PR TITLE
Esmf810

### PR DESCRIPTION
**Pull request by junwang-noaa**
## Description

-Add GFSv16 performance tuning to fv3 develop branch.
-rt_orion.conf is merged into rt.conf, and orion baseline is moved to emc.nemspara account.
The baseline results will change because the isrctermprocessing is changed from 1 to 0, this will allow a more balanced regridding between fcst grid and write grid.

### Issue(s) addressed

[ufs-community/ufs-weather-model#142](https://github.com/ufs-community/ufs-weather-model#142)

## Testing

How were these changes tested?
The RT test has been run on dell, cray, orion, is now running on cray.

What compilers / HPCs was it tested with?
Intel compiler.
Are the changes covered by regression tests?
Yes.

## Dependencies

[fv3atm PR #160](https://github.com/NOAA-EMC/fv3atm/pull/160)
[nems PR #76](https://github.com/NOAA-EMC/NEMS/pull/76)

These two PRs need to be committed first.

### CODE REVIEW

File: conf/configure.fv3.wcoss_cray [hunk 1]
```diff
@@ -48,7 +48,7 @@ CFLAGS := $(INCLUDE)
 FFLAGS := $(INCLUDE) -fno-alias -auto -safe-cray-ptr -ftz -assume byterecl -nowarn -sox -align array64byte
 
 CPPDEFS += -Duse_libMPI -Duse_netCDF -DSPMD -DUSE_LOG_DIAG_FIELD_INFO  -DUSE_GFSL63 -DGFS_PHYS -Duse_WRTCOMP
-CPPDEFS += -DNEW_TAUCTMAX -DINTERNAL_FILE_NML -DNO_PARALLEL_NETCDF
+CPPDEFS += -DNEW_TAUCTMAX -DINTERNAL_FILE_NML
```
Comment (DusanJovic-NOAA) [inline conf/configure.fv3.wcoss_cray:51]: if PARALLEL_NETCDF is now enabled on Cray, add:
`set(PARALLEL_NETCDF ON  CACHE BOOL "Enable parallel NetCDF" FORCE)`
in `cmake/configure_wcoss_cray.cmake`, similar to configure_wcoss_dell_p3.cmake, for example

**Comment by climbfuji** [comment]
@junwang-noaa I am going through my 777 (!) emails while I was on leave, then look at this PR first thing.

File: tests/rt.sh [hunk 2]
```diff
@@ -355,11 +355,6 @@ SINGLE_NAME=''
 
 TESTS_FILE='rt.conf'
 
-if [[ $MACHINE_ID = orion.* ]]; then
```
Comment (climbfuji) [inline tests/rt.sh:358]: Should we do the same with Cheyenne and Stampede in the future?

**Review by climbfuji — COMMENTED** [review]
Looks good so far, I am compiling the software stack for hera.gnu and will run the tests (including the parallel tests) afterwards.

File: tests/rt.sh (hunk 2 above)
Comment (junwang-noaa) [reply tests/rt.sh:358]: I think so, but I think we may need an option to define which tests we will not run on a certain platforms.

Comment (climbfuji) [reply tests/rt.sh:358]: Yes, I remember we talked about this. Something like a `-cheyenne.intel`, i.e. the `-` negates running the test on that platform. Same for `COMPILE` and `RUN` lines. Not that for compile steps that work on all platforms, you can just use one `COMPILE` line and leave the machine column empty.

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]

**Pull request merged**